routes/extensionRequests.js

```diff
@@ -18,7 +19,7 @@ router.get("/:id", authenticate, authorizeRoles([SUPERUSER, APPOWNER]), extensio
 router.patch(
   "/:id",
   authenticate,
-  authorizeRoles([SUPERUSER, APPOWNER]),
+  skipIfDev(authorizeRoles([SUPERUSER, APPOWNER])),
```

Comment:
1. skipIfDev does not describe what it does by its name.

Reply:
updated the name. Please have a look